docs/index.mdx

```diff
@@ -16,27 +16,46 @@ SCS is built, backed, and operated by an active open-source community worldwide.
 
 ## Use Cases and Deployment Examples
 
-### IaaS Layer
+### Virtualization (IaaS) Layer
+
+The SCS IaaS reference implementation is based on [OSISM](https://osism.tech/).
 
 #### Quick Start with Cloud-In-A-Box
 
-The fastest way to get in touch with SCS is to deploy a SCS cloud virtually. The Cloud-In-A-Box was built explicitly for this scenario. Check it out [here](/docs/iaas/guides/deploy-guide/examples/cloud-in-a-box)
+You can do a single node installation for learning, testing or development purposes.
+The Cloud-In-A-Box configuration was built explicitly for this scenario.
+Check it out [here](/docs/iaas/deployment-examples/cloud-in-a-box)
+It comes with a complete set of services, even ceph is part of it
+(despite of course not offering much redundancy on a single-node).
 
 #### Reference Implementation Testbed
 
+The fastest way to get in touch with SCS is to deploy a SCS cloud virtually.
+
 This means that you set up an SCS test installation including all the infrastructure
 pieces such as database, message queueing, ceph, monitoring and logging, IAM, the
 [OpenStack](https://openstack.org/) core services, and (soon) the Container layer
-on top of an existing IaaS platform.
+on top of an existing OpenStack IaaS platform, ideally one that allows for nested
+virtualization.
 
-The SCS IaaS reference implementation is based on [OSISM](https://osism.tech/). Read on the
-[OSISM testbed docs](https://docs.osism.de/testbed/) to learn how to get the
+Read the [testbed docs](/docs/iaas/deployment-examples/testbed) to learn how to get the
 testbed running. Please read carefully through the
-[deployment](https://docs.osism.de/testbed/deployment.html) section of the
+[deployment](https://osism.tech/docs/testbed#deployment) section of the
```

Comment:
I think it makes more sense to use ` /docs/iaas/deployment-examples/testbed#deployment` here. This way we do not link to an external document.

Reply:
Agree, thanks!
Addressed in 9fc2e2fe.